test/forms/mappings/SelectLocalDateFormatterSpec.scala

```diff
@@ -143,40 +156,52 @@ class SelectLocalDateFormatterSpec extends SpecBase {
     val day: Int = 1
 
     val key = "start"
-    val invalidMsgKey = "cf.form.error.start.date-number-invalid"
-    val monthMsgKey = "cf.form.error.start.date.invalid.month"
-    val yearMsgKey = "cf.form.error.start.date.invalid.year"
-    val invalidRealDateMsgKey = "cf.form.error.start.date.invalid.real-date"
+    val emptyMonthAndYearMsgKey = "cf.cash-account.transactions.request.start.date.empty.month.year"
+    val emptyMonthMsgKey = "cf.cash-account.transactions.request.start.date.empty.month"
+    val emptyYearMsgKey = "cf.cash-account.transactions.request.start.date.empty.year"
+    val invalidDateMsgKey = "cf.cash-account.transactions.request.start.date.invalid"
 
     val bindDataValid: Map[String, String] = Map(
       "start.day" -> day.toString, "start.month" -> "10", "start.year" -> "2022")
 
     val bindDataEmptyDate: Map[String, String] = Map(
```

Comment:
Could we remove this please bindDataEmptyDate, this currently is not being used anywhere? 
Could you also please remove them as they are also not being used:

val year1
val bindDataInValidDate